content/applications/sales/sales/products_prices/prices/currencies.rst

```diff
@@ -1,49 +1,236 @@
-    select it in the list and press *Activate* in the top-right corner. Now it will show up in
-    currencies drop-down lists.
+   To ensure currency rates are updated automatically, enable the *automatic currency rates* feature
+   on the *Accounting* settings page (:menuselection:`Accounting app --> Configuration --> Settings
+   --> Currencies section`).
+
+   .. image:: currencies/automatic-currency-rates.png
+      :align: center
+      :alt: How the main currency feature appears on settings page in Odoo Accounting.
 
-Prices in foreign currencies can be defined in two fashions.
+   Click the checkbox beside the :guilabel:`Automatic Currency Rates` feature, choose a designated
+   bank to get the currency rates from in the :guilabel:`Service` field drop-down menu, and select
+   an :guilabel:`Interval` of time for the updates to take place. Then determine when the date of
+   the :guilabel:`Next Run` should be.
 
-Automatic conversion from public price
-======================================
+   To instantly update the currency rates, click the :guilabel:`🔁 (circular arrows)` icon, located
+   to the right of the :guilabel:`Next Run` field.
 
-The public price is in your company's main currency (see
-:menuselection:`Accounting --> Settings`) and is set in product detail form.
+   When all configurations are complete, be sure to :guilabel:`Save` all changes.
 
-.. image:: currencies/public_price.png
+.. note::
+   All payment methods **must** be in the same currency as the sales journal, or the company
+   currency, if the company currency is not set. If it is not the same, a :guilabel:`Validation
+   Error` message appears.
+
+View, edit, and add currencies
+==============================
+
+To view, edit, and add currencies to the database, making them available on pricelists and on the
+:guilabel:`Main Currency` drop-down menu, click the :guilabel:`Currencies` link, located beneath the
+:guilabel:`Currency` field on the *Accounting* app's :guilabel:`Settings` page.
+
+When the :guilabel:`Currencies` link is clicked, a separate :guilabel:`Currencies` page is revealed.
+
+.. image:: currencies/main-currencies-page.png
    :align: center
+   :alt: How the main currencies page appears in Odoo Accounting.
+
+On this page, Odoo provides a master list of 167 global currencies. Each row shows the corresponding
+:guilabel:`Currency`, :guilabel:`Symbol`, :guilabel:`Name`, date of the :guilabel:`Last Update`, and
+:guilabel:`Current Rate` (compared to the default currency of the country in which the company is
+based).
+
+To the far right, there are two columns, which can be toggled on or off:
+
+- :guilabel:`Use on eBay`: this currency can be used with the connected eBay account (if
+   applicable).
+- :guilabel:`Active`: this currency is activated, which means it can be added to a pricelist, or
+   used as the main currency of the company, if desired (via :menuselection:`Accounting app -->
+   Configuration --> Settings --> Currencies section`).
```

Comment:
Bullet alignment
```suggestion
- :guilabel:`Use on eBay`: this currency can be used with the connected eBay account (if
  applicable).
- :guilabel:`Active`: this currency is activated, which means it can be added to a pricelist, or
  used as the main currency of the company, if desired (via :menuselection:`Accounting app -->
  Configuration --> Settings --> Currencies section`).
```